DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Send/RhinoRootObjectBuilder.cs

```diff
@@ -17,10 +19,12 @@ namespace Speckle.Connectors.Rhino7.Operations.Send;
 public class RhinoRootObjectBuilder : IRootObjectBuilder<RhinoObject>
 {
   private readonly IUnitOfWorkFactory _unitOfWorkFactory;
+  private readonly ISendConversionCache? _sendConversionCache;
 
-  public RhinoRootObjectBuilder(IUnitOfWorkFactory unitOfWorkFactory)
+  public RhinoRootObjectBuilder(IUnitOfWorkFactory unitOfWorkFactory, ISendConversionCache? sendConversionCache = null)
```

Comment:
If the concrete type of this interface not registered at all in Connector, this might throw but not sure. @BovineOx can put more input on this. Do we need nullability here since this is connector specific builder?

Reply:
Good point actually. If we know Rhino's going to have it, it doesn't make sense here for it to be nullable 🤔

Reply:
done, we now have a null send conversion cache for future use, and it's not optional anywhere now